exec/java-exec/src/test/java/org/apache/drill/exec/udf/dynamic/TestDynamicUDFSupport.java

```diff
@@ -104,8 +104,10 @@ public static void buildAndStoreDefaultJars() throws IOException {
   @Before
   public void setupNewDrillbit() throws Exception {
     udfDir = dirTestWatcher.makeSubDir(Paths.get("udf"));
+    File udfLocalDir = dirTestWatcher.makeSubDir(Paths.get("udf", "local"));
```

Comment:
The `DirTestWatcher` has internal support for each of Drill's working directories. Might we want to add another directory for UDF files?

Reply:
Nice set of improvements in setting up the UDF dir. Looks very clean.

Should this code then use, say, `dirTestWatcher.getUdfDir()`? Or,

```
File udfLocalDir = new File(dirTestWatcher.getUdfDir(), "local");
```

Or, since the UDF code works from the locations set in the conf file, should this be something like:

```
File udfLocalDir = new File(config.getString(ExecConstants.UDF_DIRECTORY_LOCAL));
```

Reply:
Thanks, replaced it with
```
File udfLocalDir = new File(dirTestWatcher.getUdfDir(), "local");
```